openedx/core/djangoapps/credentials/tests/factories.py

```diff
@@ -28,4 +28,4 @@ class UserCredential(DictFactoryBase):
     status = 'awarded'
     uuid = factory.Faker('uuid4')
     certificate_url = factory.Faker('url')
-    credential = factory.LazyFunction(partial(generate_instances, ProgramCredential, count=1))
+    credential = ProgramCredential()
```

Comment:
Should this perhaps be a subfactory instead, rather than an actual programcredential?

Reply:
Yes absolutely. Whoops.

Reply:
Oh wait, that is the name of the factory above in that same module. So I think it's fine.